doc/sphinx/source/projects/adt7420_console/index.rst

```diff
@@ -0,0 +1,5 @@
+.. toctree::
```

Comment:
Project heading missing

Add:
ADT7420 Console Application
===================

Reply:
![image](https://github.com/analogdevicesinc/precision-converters-firmware/assets/118167755/f098240c-66cc-4514-9432-8e08b6a95e77)

It shoes <no title> in the html page. I think it should be fixed with this